scripts/jest/setup.ts

```diff
@@ -60,6 +60,7 @@ const windowGuardian = {
 };
 
 // Stub global Guardian object
+// @ts-ignore
 window.guardian = windowGuardian;
```

Comment:
we shouldn't be allowed to do this, but know its only in the case of Jest, I think its fine to ignore

Reply:
I think it's worth commenting here why, or possibly casting to the required type (with comment) as @ts-ignore is a bit cryptic.